### DISCUSSION TITLE
How to color bullet icon

### DISCUSSION THREAD

**Discussion by 2rohith2** [ueberdosis/tiptap#4302] [General Questions]
My usecase, I need to add color to the bullet icons by combining [color plugin](https://tiptap.dev/api/extensions/color) & [bullet plugins](https://tiptap.dev/api/nodes/bullet-list).

I am able to combine both, the support I need is that when the whole line is selected I need to add the selected color to the bullet icon.

P.S: Observe the 2nd bullet point in the screenshot, I need to add color like this, to the bullet icon when whole line is selected with chosen the color.

How add color to the bullet icons when whole bullet list is selected with the chosen color ?

<img width="285" alt="image" src="https://github.com/ueberdosis/tiptap/assets/5773985/cf570b09-f809-467c-bc6d-eb99cdc1b115">

**Comment by fundlg** [comment]
@2rohith2 Hey, I'm facing the same challenge, did you succeed with that?
